package.json

```diff
@@ -8,8 +8,8 @@
     "test": "test"
   },
   "scripts": {
-    "test": "gulp test",
-    "start": "gulp watch"
+    "test": "./node_modules/.bin/mocha test/*",
```

Comment:
Missing `--reporter nyan` - clear regression compared to current test command (https://github.com/netguru/eslint-plugin-ember/pull/4/files#diff-b9e12334e9eafd8341a6107dd98510c9L29).